### PR TITLE
vlan change enhancement

### PR DESCRIPTION
**Pull request by tjjh89017**
change vlan usage
related to https://github.com/harvester/harvester-installer/pull/278

### CODE REVIEW

**Comment by bk201** [comment]
We need to wait until https://github.com/harvester/docs/pull/192 is merged.
The new framework has a better muLti-version support and this change should be in v1.1

**Comment by bk201** [comment]
#192 is in. Please rebase on the new framework. Thanks!

**Comment by tjjh89017** [comment]
rebase to latest docs version

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

File: docs/install/harvester-configuration.md [hunk 1]
```diff
@@ -91,6 +91,20 @@ For instance, if you have in your Harvester Configuration file specified `os.hos
...
+
+The version of scheme is reserved for the future configuration migration.
+
+This configuration is mandatory for migrating configuration to new scheme version. It tells the Harvester the previous version and need to migrate.
+
+!!!note
+    This field didn't take any effect in the current Harvester version.
+
+!!!warning
+    Make sure that your custom configuration always has the correct scheme version.
```
[the remark this answers is not in the document]
Comment (vickyhella) [reply docs/install/harvester-configuration.md:106]: Please use Docusaurus admonitions such as `:::note` and `:::caution`

File: docs/install/harvester-configuration.md [hunk 2]
```diff
@@ -91,6 +91,20 @@ For instance, if you have in your Harvester Configuration file specified `os.hos
 
 :::
 
+### `scheme_version`
+
+#### Definition
+
+The version of scheme is reserved for the future configuration migration.
+
+This configuration is mandatory for migrating configuration to new scheme version. It tells the Harvester the previous version and need to migrate.
```
Comment (vickyhella) [inline docs/install/harvester-configuration.md:100]:
```suggestion
This configuration is mandatory for migrating the configuration to a new scheme version. It tells Harvester the previous version and the need to migrate.
```

Comment (vickyhella) [reply docs/install/harvester-configuration.md:100]: Please double-check if I understand it correctly ^

Comment (tjjh89017) [reply docs/install/harvester-configuration.md:100]: It is correct.
thanks for correction.

File: docs/install/harvester-configuration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tjjh89017) [reply docs/install/harvester-configuration.md:106]: Ok got it.

File: docs/install/harvester-configuration.md [hunk 3]
```diff
@@ -91,6 +91,20 @@ For instance, if you have in your Harvester Configuration file specified `os.hos
 
 :::
 
+### `scheme_version`
+
+#### Definition
+
+The version of scheme is reserved for the future configuration migration.
```
[the remark this answers is not in the document]
Comment (tjjh89017) [reply docs/install/harvester-configuration.md:98]: Ok got it.

**Review by vickyhella — APPROVED** [review]
LGTM, thanks

**Pull request merged**